src/main/kotlin/org/malachite/estella/people/api/HrPartnerController.kt

```diff
@@ -34,7 +35,7 @@ class HrPartnerController(
     fun getHrPartner(@PathVariable("hrPartnerId") hrPartnerId: Int): ResponseEntity<HrPartner> =
         hrPartnerService.getHrPartner(hrPartnerId).let { ResponseEntity.ok(it) }
 
-
+    @Transactional
```

Comment:
Nie chcemy dodać tego do większej liczby endpointów, czy też wprowadzamy podejście że jak nie działa to sprawdzamy czy zacznie z tą adnotacją XDD

Reply:
Nie zapominaj, że to jest hotfix - task na to, żeby sprawdzić, czy to faktycznie jest rozwiązanie jest wciąż w Improvements and fixes.

Reply:
@Davenury A to ok XD